Jenkinsfile

```diff
@@ -48,14 +26,12 @@ void runTestSuite(String DOCKER_IMAGE_VERSION, CLIENT_VERSION, PMM_QA_GIT_BRANCH
     env.BATS_TESTS_RESULT = testSuiteJob.result
 }
 
-void runUItests(String DOCKER_IMAGE_VERSION, CLIENT_VERSION, PMM_QA_GIT_BRANCH, PMM_QA_GIT_COMMIT_HASH, PMM_SERVER_IP) {
-    e2eTestJob = build job: 'pmm2-ui-tests', propagate: false, parameters: [
+void runUItests(String DOCKER_IMAGE_VERSION, CLIENT_VERSION, PMM_QA_GIT_BRANCH, PMM_QA_GIT_COMMIT_HASH) {
+    e2eTestJob = build job: 'pmm2-ui-tests-temp', propagate: false, parameters: [
```

Comment:
@puneet0191 Looks good. Do we want to have a job name that suggests this is a permanent change rather than a `temp` solution?

Reply:
I'd suggest we do keep around pipelines we do not use or no longer use, since they make it difficult to find the right one.

Reply:
@atymchuk yes, we generally disable pipelines after using them for testing purposes, as you can see in my PR description, I am going to remove the `temp` and change it back, I am only using it for triggering the FB execution.